src/index.js

```diff
@@ -212,12 +212,10 @@ class ServerlessIOpipePlugin {
   checkToken() {
     const token = this.getOptions().token;
     if (!token) {
-      this.track({
-        action: 'token-missing'
-      });
-      throw new Error(
-        'No iopipe token found. Specify "iopipeToken" in the "custom" object in serverless.yml.'
-      );
+      const msg =
+        'You did not specify iopipeToken in your custom section of serverless.yml. iopipe will fall back to $IOPIPE_TOKEN in the lambda environment';
```

Comment:
I want to leave this for now, but look out in the future in case users find this noisy.